# HG changeset patch
# User Thomas Treichl <Thomas.Treichl@gmx.net>
# Date 1234936068 18000
#      Wed Feb 18 00:47:48 2009 -0500
# Node ID d943e26d30e2c27eaeb7a79b1583d03a03e08ce4
# Parent  a013ff655ca48f15cc2c72aa938f2e3cb6e50ba9
Add documentation for configure options	"--without-framework-carbon" and "--without-framework-opengl".

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2009-02-17  Thomas Treichl  <Thomas.Treichl@gmx.net>
+
+	* interpreter/install.txi: Add documentation for configure options
+	"--without-framework-carbon" and "--without-framework-opengl".
+
 2009-02-17  Jaroslav Hajek  <highegg@gmail.com>
 
 	* interpreter/arith.txi: Add reference to "cummin" and "cummax".
 
 2009-02-11  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/Makefile.in (uninstall): Use $(DESTDIR) here too.
 	Uninstall Info dir file.  Uninstall DOC file.
diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -144,16 +144,29 @@ Octave.  By default, configure first loo
 libraries on your system, including optimized BLAS implementations such
 as the free ATLAS 3.0, as well as vendor-tuned libraries.  (The use of
 an optimized BLAS will generally result in several-times faster matrix
 operations.)  Only use this option if your system has BLAS/LAPACK
 libraries that cause problems for some reason.  You can also use
 @code{--with-blas=lib} to specify a particular BLAS library
 @code{-llib} that configure doesn't check for automatically.
 
+@item --without-framework-carbon
+Don't use framework Carbon headers, libraries and specific source code
+for compilation even if the configure test succeeds (the default value
+is @code{--with-framework-carbon}). This is a platform specific configure
+option for Mac systems.
+
+@item --without-framework-opengl
+Don't use framework OpenGL headers, libraries and specific source code
+for compilation even if the configure test succeeds. If this option is
+given then OpenGL headers and libraries in standard system locations are
+tested (the default value is @code{--with-framework-opengl}). This is a
+platform specific configure option for Mac systems.
+
 @item --help
 Print a summary of the options recognized by the configure script.
 @end table
 
 See the file @file{INSTALL} for more information about the command line
 options used by configure.  That file also contains instructions for
 compiling in a directory other than where the source is located.
 
